.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint && yarn tsc
```

Comment:
Perhaps this should be in `pre-commit`? or maybe pre-push isn't necessary?
The reason I ask is because you'd ideally want it to lint/warn about issues prior to committing.

Reply:
The idea was to run prettier --write on pre-commit and prettier --check on pre-push but I see your point 🤔

Reply:
I basically copied this pattern from [api3-dao-dashboard](https://github.com/api3dao/api3-dao-dashboard/blob/main/.husky/pre-push) so maybe @Siegrift can help out here.

Reply:
Please, do not run `prettier --write` on pre-commit or anywhere else for that matter automatically. The developers should be always aware of the changes they're committing.

Reply:
I'm confused 🤔 Isn't Airnode and all other projects doing exactly that already? https://github.com/api3dao/airnode/blob/master/.husky/pre-commit

Reply:
That's only for the validator autogenerated docs. For the rest it's calling `yarn lint` and that's calling `yarn prettier:check` which is just a check showing the errors, not fixing them automatically.

Reply:
Right, but there are still multiple other repos where we are doing `--write` like in [operations](https://github.com/api3dao/operations/blob/main/.husky/pre-commit#L3), [services](https://github.com/api3dao/services/blob/main/.husky/pre-commit#L0-L1), etc. I wonder if we should make this consistent at across repos.

Reply:
I think we should and IMO it's not a good idea to have an automatic write. But I don't want to force it unless others agree.

Reply:
Removed --write and pre-push hock